lib/exercises.rb

```diff
@@ -2,19 +2,44 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n2) .sort & .join on each word is its own O(n)?
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
You have n words and if each word is approx m letters, this would be O(n * m log m) since you're sorting each word.